service/src/main/java/uk/gov/hmcts/reform/fpl/service/DraftCMOService.java

```diff
@@ -80,16 +89,35 @@ public DynamicList buildDynamicListFromHearingDetails(List<Element<HearingBookin
         return DynamicList.toDynamicList(hearingDates, DynamicListElement.EMPTY);
     }
 
-    public CaseManagementOrder getCaseManagementOrder(CaseDetails caseDetails) {
+    public CaseManagementOrder prepareCMO(CaseDetails caseDetails) {
         DynamicList list = mapper.convertValue(caseDetails.getData().get("cmoHearingDateList"), DynamicList.class);
+        CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
         return CaseManagementOrder.builder()
             .hearingDate(list.getValue().getLabel())
             .id(list.getValue().getCode())
+            .directions(combineAllDirectionsForcmo(caseData))
             .build();
     }
 
     private String formatLocalDateToMediumStyle(LocalDate date) {
         return dateFormatterService.formatLocalDateToString(date, FormatStyle.MEDIUM);
     }
+
+    // Temporary, to be replaced by directionHelperService.combineAllDirections once all directions have been added
+    private List<Element<Direction>> combineAllDirectionsForcmo(CaseData caseData) {
+        List<Element<Direction>> directions = new ArrayList<>();
```

Comment:
aren't all directions present now?

Reply:
Not yet, missing two. Hoping to merge this and remove